src/main/resources/META-INF/resources/primefaces/inputnumber/1-inputnumber.js

```diff
@@ -127,11 +127,6 @@ PrimeFaces.widget.InputNumber = PrimeFaces.widget.BaseWidget.extend({
     },
 
     getValue: function () {
-        var val = this.autonumeric.getNumericString();
-        if (val && this.cfg.decimalPlaces) {
-            return parseFloat(val).toFixed(this.cfg.decimalPlaces);
-        }
-
-        return val;
+        return this.autonumeric.getNumericString();
     }
```

Comment:
@mertsincan You just fixed this to do this please review.

https://github.com/primefaces/primefaces/issues/5402

Reply:
Ok ! I'll check it. Thanks a lot!